fastcrypto-tbls/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "fastcrypto-tbls"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+readme = "README.md"
+description = "Threshold BLS and DKG protocols"
+repository = "https://github.com/MystenLabs/fastcrypto"
+
+[dependencies]
+fastcrypto = { path = "../fastcrypto" }
```

Comment:
i think this needs to be a a published version of fastcrypto (you can try `cargo publish -p fastcrypto-tbls` --dry-run to confirm)